### DISCUSSION TITLE
Button More…

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#247] [Q&A] [answered]
In version 3.0.25 I miss the "More" button under which you can click the "Show on Google Maps" button.
Maybe it's hidden somewhere else?

Otherwise a big compliment.
Cruiser just keeps getting better!
But also a criticism: The whole "Settings"  needs to be cleaned up thoroughly. It's often confusing...

**Comment by devemux86** [comment]
> I miss the "More" button under which you can click the "Show on Google Maps" button.

The "More" was removed, it is better to have all actions in 1 list for direct and easy access.

Does anyone use the "Show on Google Maps" with Cruiser and on what map element?
I can return it if you need it.

> Otherwise a big compliment.
Cruiser just keeps getting better!

@Leeb64 many thanks! :slightly_smiling_face: 

>  But also a criticism: The whole "Settings" needs to be cleaned up thoroughly. It's often confusing...

They are organized into groups, most users can stick with the default settings without changing anything or going into the settings at all.

Are there any specific areas that need better organization or descriptions?

**Reply by WolfgangZ42** [reply]
> Does anyone uses the "Show on Google Maps" with Cruiser

Didn't use it so far. And via "Favorite" you can get the coordinates for further processing - for my opinion fair enough ...

**Reply by WolfgangZ42** [reply]
For those who need a detailled description:
![2023-07-20 09 14 03](https://github.com/devemux86/cruiser/assets/121806238/57991c69-b9d0-4155-892c-a8a95242e1e6)  ![2023-07-20 09 14 09](https://github.com/devemux86/cruiser/assets/121806238/6b5c35f1-07ec-4a95-ab3a-2655c2686db2)  ![2023-07-20 09 15 56](https://github.com/devemux86/cruiser/assets/121806238/08e264eb-ddbc-4c02-b221-5275f202c3df)  ![2023-07-20 09 21 08](https://github.com/devemux86/cruiser/assets/121806238/a21927cd-88be-41ab-8f4f-123a5a67f62e)

**Comment by ramondevic** [comment]
Yes, I use it.

I don’t use “mostrar dirección” (show address),

 

 

De: Leeb64 ***@***.***> 
Enviado el: jueves, 20 de julio de 2023 8:09
Para: devemux86/cruiser ***@***.***>
CC: Subscribed ***@***.***>
Asunto: [devemux86/cruiser] Button : More (Discussion #247)

 

In version 3.0.25 I miss the "More" button under which you can click the "Show on Google Maps" button.
Maybe it's hidden somewhere else?

Otherwise a big compliment.
Cruiser just keeps getting better!
But also a criticism: The whole adjustment shop needs to be cleaned up thoroughly. It's often confusing...

**Comment by ramondevic** [comment]
When you click on a favorite on the map, there is no option to see the “coordinates”, nor to “see google maps”, only the “show direction” and I don't know if anyone used it because it doesn't seem very practical.

 

 

De: Emux ***@***.***> 
Enviado el: jueves, 20 de julio de 2023 8:27
Para: devemux86/cruiser ***@***.***>
CC: Subscribed ***@***.***>
Asunto: Re: [devemux86/cruiser] Button : More (Discussion #247)

 

I miss the "More" button under which you can click the "Show on Google Maps" button.

The "More" was removed, it is better to have all actions in 1 list for direct and easy access.

Does anyone uses the "Show on Google Maps" with Cruiser?
I can return it if you need it.

Otherwise a big compliment.
Cruiser just keeps getting better!

@Leeb64 <https://github.com/Leeb64>  many thanks! 🙂

But also a criticism: The whole "Settings" needs to be cleaned up thoroughly. It's often confusing...

They are organized into groups, most users can stick with the default settings without changing anything or going into the settings at all.

Is there any specific area that needs better organization or descriptions?

**Comment by ramondevic** [comment]
And if you click on "share" this comes out, I would prefer to be able to choose if I want the coordinate or Google Maps.

I don't know if anyone uses "plus.codes" or "w3w.co"

 

https://osm.org/go/xUfGg2j0--?m

https://plus.codes/8FH4W6HX+6J

https://w3w.co/sentí.ducha.enviar

 

I don't know what other users think.

 

 

 

De: Ramon Amblàs ***@***.***> 
Enviado el: jueves, 20 de julio de 2023 9:00
Para: 'devemux86/cruiser' ***@***.***>; 'devemux86/cruiser' ***@***.***>
CC: 'Subscribed' ***@***.***>
Asunto: RE: [devemux86/cruiser] Button : More (Discussion #247)

 

When you click on a favorite on the map, there is no option to see the “coordinates”, nor to “see google maps”, only the “show direction” and I don't know if anyone used it because it doesn't seem very practical.

 

 

De: Emux ***@***.*** ***@***.***> > 
Enviado el: jueves, 20 de julio de 2023 8:27
Para: devemux86/cruiser ***@***.*** ***@***.***> >
CC: Subscribed ***@***.*** ***@***.***> >
Asunto: Re: [devemux86/cruiser] Button : More (Discussion #247)

 

I miss the "More" button under which you can click the "Show on Google Maps" button.

The "More" was removed, it is better to have all actions in 1 list for direct and easy access.

Does anyone uses the "Show on Google Maps" with Cruiser?
I can return it if you need it.

Otherwise a big compliment.
Cruiser just keeps getting better!

@Leeb64 <https://github.com/Leeb64>  many thanks! 🙂

But also a criticism: The whole "Settings" needs to be cleaned up thoroughly. It's often confusing...

They are organized into groups, most users can stick with the default settings without changing anything or going into the settings at all.

Is there any specific area that needs better organization or descriptions?

**Reply by devemux86** [reply]
> And if you click on "share" this comes out, I would prefer to be able to choose if I want the coordinate or Google Maps.
I don't know if anyone uses "plus.codes" or "w3w.co"

We could indeed have a selection in the "Share", so users can select their desired sharing format:
- Coordinates
- OpenStreetMap link
- Google Maps link
- [Plus code](https://maps.google.com/pluscodes/)
- [what3words](https://what3words.com/)
- ...

**Comment by ramondevic** [comment]
Yes 😊

... and

*	Coordinates

 

 

De: Emux ***@***.***> 
Enviado el: jueves, 20 de julio de 2023 9:37
Para: devemux86/cruiser ***@***.***>
CC: ramondevic ***@***.***>; Comment ***@***.***>
Asunto: Re: [devemux86/cruiser] Button : More (Discussion #247)

 

And if you click on "share" this comes out, I would prefer to be able to choose if I want the coordinate or Google Maps.
I don't know if anyone uses "plus.codes" or "w3w.co"

We could indeed have a selection in the "Share", so users can select their desired sharing format:

*	OpenStreetMap link
*	Google Maps link
*	Plus code <https://maps.google.com/pluscodes/> 
*	what3words <https://what3words.com/> 
*	...

**Comment by Cer-o** [comment]
> Does anyone use the "Show on Google Maps" with Cruiser and on what map element?
> I can return it if you need it.

Ich nutze diese Ressource und würde sie gerne haben, deshalb ist meine Stimme JA FÜR GOOGLE MAPS

**Reply by Leeb64** [reply]
the same applies to me

**Comment by Ciutnik** [comment]
I would also like to ask you to restore the ability to show the position in Google Maps (Street View) and OpenStreetMap. Seeing obvious routing errors, I often correct the OSM map directly from Cruiser (previously K1/K2). Of course, it can be done differently, but the direct link was convenient.

I even wanted to propose the use of geographic coordinates shown in the window after a long press on the map. For now, their (long)touch is not used.

<img src="https://github.com/devemux86/cruiser/assets/122647782/e9ec0f96-e571-4b57-b61e-6d64b2b1cca8" width="50%"/>

**Reply by devemux86** [reply]
> I even wanted to propose the use of geographic coordinates shown in the window after a long press on the map.

The coordinates already appear in the window title, like in your image.

Or do you mean the format of the coordinates?
This can be changed in "Settings | Application | Advanced | Coordinates".

**Comment by DolomitesTom** [comment]
> desDoes anyone use the "Show on Google Maps" with Cruiser and on what map element?
I can return it if you need it.

Yes, very often, I need this back too.

**Comment by Ciutnik** [comment]
@devemux86 
> The coordinates already appear in the window title, like in your image.

Of course! I meant some action on press/long press coordinates. You could open a new window with actions like "Share", "Open in Google Maps", "Open in OpenStreetMap", or maybe even something I know from another app: "Show what's nearby", which lists all POIs with bearing and distance.

**Reply by devemux86** [reply]
> You could open a new window with actions like "Share", "Open in Google Maps", "Open in OpenStreetMap"

This is the plan. :slightly_smiling_face: 

> "Show what's nearby", which lists all POIs with bearing and distance.

This is now possible with the offline search.
It displays the POI inside the screen bounds with direction + distance, sorted by distance from user location or map center.

Or do you mean search and find all POI regardless their category?
It is possible, but these will be too many poi.

**Reply by Ciutnik** [reply]
> Or do you mean search and find all POI regardless their category?

Yes. That's what I meant because I know it from another app. The search circle is really small (maybe 50m), but in addition to POIs, it also shows addresses, found businesses, services, etc. It's not a key feature right now, but maybe it's an idea for the future.

**Reply by 1maxi2** [reply]
"Nutzt jemand die Funktion „Auf Google Maps anzeigen“ mit Cruiser und auf welchem ​​Kartenelement?
Ich kann es bei Bedarf zurückgeben."

Ja ich habe das sehr oft benutzt und würde das gerne haben.

**Reply by r11d11** [reply]
I would also be happy if the button "Show in Google Maps" returns

**Comment by devemux86** [comment]
"More…" functions will return, I need to check some things. :slightly_smiling_face:

**Comment by devemux86** [comment]
The button More will be available again in the next Cruiser update:
- Show address
- Share
- Show on OpenStreetMap
- Show on Google Maps
- Show on Mapillary

The "Share" options are described below: https://github.com/devemux86/cruiser/discussions/247#discussioncomment-6529407

**Reply by DolomitesTom** [reply]
My preference would be share as plus codes. The others I don't use at all.

**Reply by devemux86** [reply]
@DolomitesTom Share or Copy?

(they are different actions)

**Reply by Ciutnik** [reply]
OsmAnd under "Share" proposes "geo:", which allows you to transfer the location to other map applications. Maybe something like this makes sense?

**Comment by DolomitesTom** [comment]
> Share or Copy?

 I've meant share.

**Comment by devemux86** [comment]
> OsmAnd under "Share" proposes "geo:", which allows you to transfer the location to other map applications. Maybe something like this makes sense?

The new "Share" menu in Cruiser will allow to copy / share any of these:
- OpenStreetMap
- Google Maps
- Plus code
- what3words
- Coordinates
- geo:

**Comment by Ciutnik** [comment]
Will it be possible to share the location FROM Google Maps TO Cruiser? Since _Map2Geo_ doesn't work, I use Locus (from Google Maps to Locus, from Locus to Cruiser). Reliable, though moderately comfortable.

**Reply by devemux86** [reply]
@Ciutnik do you mean parsing Google Maps links or something else?

Google Maps uses many domains and often changes the urls, it requires a lot of work to maintain. :slightly_smiling_face:

**Reply by Ciutnik** [reply]
Yes, I forgot to add that I mean links created by Maps. They look like this (currently): https://goo.gl/maps/............... or https://maps.app.goo.gl/................. But since it requires a lot of work, it's a waste of your time. At least for me, I don't use it very often, and there are several ways to deal with it.

Edit: I've noticed that Plus Code works very well here. That's enough 👍

**Reply by devemux86** [reply]
Cruiser supports both types of plus codes:
- Local plus codes: "V75V+8Q Paris, France"
- Global plus codes: "8FW4V75V+8Q"

**Reply by 1maxi2** [reply]
Mit der App "transfer to other map "
funktioniert das übertragen vom Standort vom Google map zu Cruiser einwandfrei !

**Reply by devemux86** [reply]
Cruiser currently can open the following urls:
- OpenStreetMap (various types)
- osm.org shortlinks
- geo:
- Plus codes (also in Search)
- what3words (also in Search)

**Reply by Ciutnik** [reply]
@1maxi2 
> Mit der App "transfer to other map "

Wir meinen wahrscheinlich die gleiche Anwendung, die leider auf neueren Androids nicht mehr funktioniert.

We probably mean the same application that unfortunately no longer works on newer Androids.

**Reply by devemux86** [reply]
> We probably mean the same application that unfortunately no longer works on newer Androids.

Newer Androids have another restriction: the apps can register urls for exclusive use...

Google Maps app now probably has priority for all Google urls.

**Comment by devemux86** [comment, marked as answer]
An improved "More" button is available again in [Cruiser 3.0.27](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by DolomitesTom** [comment]
all sharing menu entries via the more button are not working on my device... nothing happes after sub-menu entry is tapped.

![grafik](https://github.com/devemux86/cruiser/assets/121371629/85b2bd6c-6dff-42e8-aede-f6c5d2db5319)

**Reply by devemux86** [reply]
@DolomitesTom what exactly does not work?

Sharing menus copy the relevant link to the clipboard,
then the Android sharing screen opens automatically:

(press Android share button)

![Share](https://github.com/devemux86/cruiser/assets/3484020/c97ac319-805e-43b2-bf70-45b09a879314)

**Reply by DolomitesTom** [reply]
link is obviously copied into the clipboard, this works, but no share menu is opened. This picture is not displayed.

**Reply by devemux86** [reply]
@DolomitesTom what Android version and device?

**Reply by DolomitesTom** [reply]
Samsung S8+ SM-G965F, Android 10 One UI 2.5, not yet tested with my other device.

**Comment by DolomitesTom** [comment]
... also with Samsung A52S 5G with Android 13 only a message occurs "copied to clipboard", but no share menu

**Comment by Leeb64** [comment]
I have the worm too...
First the Google Maps didn't open...then the app crashed...then it worked all of a sudden...then the offline maps + map style were gone again and the app froze...
Now it works properly again with Google Maps(Show on Google Maps) Strange...

**Reply by devemux86** [reply]
> Now it works properly again with Google Maps.

@Leeb64 do you mean the share Google Maps link in Cruiser?

**Reply by Leeb64** [reply]
After switching off the APP and switching it on again, offline maps + map style were gone again

**Reply by Leeb64** [reply]
->Show on Google Maps<-

**Reply by devemux86** [reply]
> After switching off the APP and switching it on again, offline maps + map style were gone again

For this, please see:
- https://github.com/devemux86/cruiser/discussions/262

**Comment by devemux86** [comment]
[Cruiser 3.0.28](https://play.google.com/store/apps/details?id=gr.talent.cruiser) was released on Google Play:
- Copies the shared link to the clipboard to use it everywhere.

- Informs Android to open the shared link with compatible apps:
Android's "Open with" dialog appears, for some links it opens directly Google maps or browser.

**Comment by DolomitesTom** [comment]
Button more works now but when I want to share plus Codes, then I want to select with which app for example WhatsApp to a colleague, but instead it opens directly Google Maps. Can it  be changed that it asks which app I want to use for sharing?

**Reply by devemux86** [reply]
> when I want to share plus Codes, then I want to select with which app

In new Androids:
Plus codes are managed by Google and they have registered Google Maps app / website as official for plus codes.

You may be able to go to Android app settings for Google Maps and uncheck the "Open supported links",
to allow other apps to use such links managed by Google?

**Reply by devemux86** [reply]
Cruiser copies the shared links to the clipboard anyway. Open any messenger and paste the link.

In new Androids the "Open with" is extremely restricted. Do not expect it to work anymore.
It just opens automatically the default associated app or the browser.

That's why I prefer the previous implementation where we only copy the links.
(not try to open any apps)

**Reply by DolomitesTom** [reply]
Ok, can live with this 😄
